signac/common/config.py

```diff
@@ -6,9 +6,6 @@
 import logging
 import os
 
-from deprecation import deprecated
-
-from ..version import __version__
```

Comment:
Can we get this change merged directly onto `next` to satisfy flake8 / pre-commit CI? Did we merge something with CI failing?

Reply:
No, just an oversight in the last rebase. I fixed it there.